### PR TITLE
Support GCS Client Libraries

### PR DESCRIPTION
**Pull request by elharo**
@akerekes

### CODE REVIEW

**Comment by chanseokoh** [comment]
This has no UI change (i.e., no GCS library in the wizards and _Add Libraries..._. Is this intended?

**Comment by elharo** [comment]
Yes, the UI appears in Eclipse Add Libraries > Google API Libraries:

image

**Comment by chanseokoh** [comment]
Ah, it's under a different menu item.

I see checking the Core library automatically checks the Google API library. However, if the Core library is automatically checked by the Storage library, the Google API isn't automatically checked:

selection_001

**Comment by elharo** [comment]
Yes, that's #1664 and we need to get this in in order to work effectively on that.

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml [hunk 1]
```diff
@@ -125,6 +125,44 @@
...
+              artifactId="joda-time"
+              groupId="joda-time"
+              version="2.9.2" />
+      </libraryFile>
+    </library>
+    
+    <library
+          id="googlecloudstorage"
+          group="clientapis"
+          name="Google Cloud Storage Client Library for Java"
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: Shouldn't `name` be externalized too?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml [hunk 2]
```diff
@@ -125,6 +125,44 @@
               version="1.22.0" />
       </libraryFile>
     </library>
+    
+    <library
+          id="googlecloudcore"
+          group="clientapis"
+          name="Google Cloud Core Client Library for Java"
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: Shouldn't `name` be externalized too?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: I think it's a proper name. Do we localize "Google Cloud Platform" for example? Let me check with the relevant team.

**Pull request merged**